src/smk/tool/geomark/tool-geomark.js

```diff
@@ -202,6 +202,14 @@ include.module( 'tool-geomark', [
                 }
             } )
 
+            // Override default handling in Identify tool
+            smk.$viewer.handlePick( 1, function ( location ) {
```

Comment:
What does this 'magic' number mean? Why is it set to 1?

Reply:
The Geomark tool collides with the Location tool when both are enabled. The location tool has priority 1. I'd suggest trying a priority of 4 for the Geomark tool.

Reply:
Thanks for finding this ... I should have also tested with all the other tools that register click handling.

A priority of 2 is enough to resolve the collision with Location, but on further thought, I'm fine with 4 - we want to help ensure there are no conflicts with other tools going forward.